htlcswitch/switch.go

```diff
@@ -2341,6 +2341,30 @@ func (s *Switch) addLiveLink(link ChannelLink) {
 	}
 	s.interfaceIndex[peerPub][link.ChanID()] = link
 
+	log.Infof("aliasmgr Calling update internal")
```

Comment:
nit: left-over info-level logs.

Reply:
:+1: will remove this with next round of fixes / final rebase